lib/htmlValidator.js

```diff
@@ -46,31 +49,43 @@ module.exports = function(app, callback) {
           'java', ['-Xss1024k', '-cp', vnu, 'nu.validator.servlet.Main', params.htmlValidator.port ? params.htmlValidator.port : '8888']
         );
 
+        console.log(('Validator intializing...').yellow);
+
         validatorProcess.stdout.on('data', (data) => {
           if (`${data}`.includes('Initialization complete')) {
+            clearTimeout(validatorTimeout);
+            console.log(('Validator initialization completed successfully.').green);
             resolve();
           }
         });
 
         validatorProcess.stderr.on('data', (data) => {
           if (`${data}`.includes('No Java runtime present, requesting install.')) {
-            console.error('error: No Java runtime detected. Check to make sure Java is installed and in your path.');
+            console.error(('There was an error in authentication. Please make sure to have Java installed.').red);
+            validatorDisabled = true;
```

Comment:
Instead of setting this to true on each error, you could set it to true in the promise rejection catch.